modules/oc-mirror-migration-process.adoc

```diff
@@ -0,0 +1,120 @@
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+storageConfig:  # REMOVE this field in v2
+  local:
+    path: /var/lib/oc-mirror
+----
++
+.Example output with oc-mirror plugin v2 configuration:
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+      - name: stable-4.17
+    graph: true
+  helm:
+    repositories:
+      - name: sbo
+        url: https://redhat-developer.github.io/service-binding-operator-helm-chart/
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+----
+
+. In oc-mirror plugin v2, mirrored content is stored in a different location than v1. You must check the cluster-resources directory inside the working directory for IDMS, ITMS, `CatalogSource`, and `ClusterCatalog` using the following command:
++
+[source,terminal]
+----
+$ ls <v2_workspace>/working-dir/cluster-resources/
+----
+
+. Once the migration is complete, verify that mirrored images and catalogs are available. Make sure that:
+- No errors or warnings occurred during mirroring.
+- No error file was generated (`working-dir/logs/mirroring_errors_YYYYMMdd_HHmmss.txt`).
+
+. Verify that mirrored images and catalogs are available using the following the commands:
++
+[source,terminal]
+----
+$ oc get catalogsource -n openshift-marketplace
+$ oc get imagedigestmirrorset,imagetagmirrorset -n openshift-config
+----
++
+For more information, refer to "Mirroring images for a disconnected installation using oc-mirror plugin v2".
+
+. [Optional] To remove images mirrored using oc-mirror plugin v1, follow these steps:
++
+[IMPORTANT]
+====
+oc-mirror plugin v2 does not automatically prune the destination registry, unlike oc-mirror plugin v1. If you want to clean up images no longer needed, use the delete functionality in v2 with `--delete-v1-images`.
+
+Once all images mirrored with oc-mirror plugin v1 are removed, you no longer need to use this flag. If you need to delete images mirrored with oc-mirror plugin v2, do not set `--delete-v1-images`.
+====
+
+.. Mirror the image using oc-mirror plugin v1.
+
+.. Upgrade the `ImageSetConfiguration` API version from `v1alpha2` (v1) to `v2alpha1` (v2), then run the following command:
++
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml file://some-dir --v2
+----
++
+[NOTE]
+====
+`storageConfig` is not a valid field in `ImageSetConfiguration and `DeleteImageSetConfiguration`. Remove this field when updating to oc-mirror plugin v2.
+====
+
+.. Generate a delete manifest and delete v1 images using the following command:
```

Comment:
Per the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command):
```suggestion
.. Generate a delete manifest and delete v1 images by running the following command:
```